scripts/src/main/resources/scripts/command/vscode

```diff
@@ -126,6 +137,32 @@ function doStartVsCode() {
   doRunVsCode "${WORKSPACE_PATH}"
 }
 
+function doAddRecommendations() {
+  local recommendations="${*}"
+  local extensionsJSON="${WORKSPACE_PATH}/.vscode/extensions.json"
+  local key="\"recommendations\": [$recommendations]"
+  if [ -f "${extensionsJSON}" ] && [ ! -s "${extensionsJSON}" ]
+  then 
+    if grep -q "\"recommendations\":" "${extensionsJSON}"
+    then
+      if ! grep -Fq "${key}" "${extensionsJSON}"
+      then
+        # if recommendations key is already in extensions.JSON
```

Comment:
Since the IDE only adds the recommendation during installation, it may not be necessary to distinguish between all three cases here. But if, for example, a user reinstalls `vscode`, this could lead to errors without distinction. I would keep it.

Reply:
Thanks for explanation. Sounds reasonable. 👍